static/style.css

```diff
@@ -189,6 +184,12 @@ kbd {
     border-radius: 6px;
 }
 
+/* highlight/mark */
+mark {
+  background-color: rgba(255, 255, 175, 0.8);
+  color: #222;
```

Comment:
I couldn't think of this, (a little lighter fg) but I think this makes it subtly nicer looking in some way, nice